valhalla/thor/worker.h

```diff
@@ -64,6 +64,7 @@ class thor_worker_t : public service_worker_t {
   std::string expansion(Api& request);
   void centroid(Api& request);
   void status(Api& request) const;
+  void expansion(Api& request) const;
```

Comment:
this is wierd, do we actually need two expansion fuctions or did you just forget to remove the old signature?

Reply:
oh no it actually looks like this one is superfluous?

Reply:
huh, wonder how that got in there.. yeah, very superfluous!